app/components/SideBar/SideBar.tsx

```diff
@@ -1,18 +1,139 @@
 'use client';
 
+import React from 'react';
+import {
+  Card,
+  Typography,
+  List,
+  Chip,
+  Accordion,
+  AccordionHeader,
+  AccordionBody,
+  Alert,
+  Input,
+} from '@material-tailwind/react';
+import {
+  PresentationChartBarIcon,
+  Cog6ToothIcon,
+  InboxIcon,
+  PowerIcon,
+  WalletIcon,
+  Squares2X2Icon,
+  CircleStackIcon,
+  ShoppingBagIcon,
+  EnvelopeIcon,
+  BriefcaseIcon,
+} from '@heroicons/react/24/solid';
+import { CubeTransparentIcon, MagnifyingGlassIcon } from '@heroicons/react/24/outline';
+import Logo from '../../assets/image/logo.svg';
 import SideBarItem from './SideBarItem';
 
-import { Table, Gear } from '@phosphor-icons/react';
+const Sidebar = () => {
+  const [openAlert, setOpenAlert] = React.useState(true);
```

Comment:
Please clean up this unused alert + commented code below

Reply:
done